src/utils.jl

```diff
@@ -415,3 +415,63 @@ macro hijack_observable(name)
     end)
 
 end
+
+############################################################
+#                                                          #
+#     Splitting Of LineString / coast lines                #
+#                                                          #
+############################################################
+
+#`LineSplitting` module defines a `split` method for vectors of `LineString` objects.
+#
+#This is needed to fix e.g. coast line displays when lon_0 is not 0 but cutting polygons at lon_0+-180.
+
+module LineSplitting
+
+	import GeoMakie.LineString
+	import GeoMakie.Observable
+	import Base.split
+	
+	function regroup(tmp::Vector)
+		coastlines_custom=LineString[]
+		println(typeof(coastlines_custom))
+		for ii in 1:length(tmp)
+			push!(coastlines_custom,tmp[ii][:]...)
+		end
+		coastlines_custom
+	end
+	
+	function split(tmp::Vector,lon0=-160.0)
```

Comment:
It would probably be useful to qualify types:
```suggestion
	function split(tmp::Vector{<:LineString}, lon0::Real)
```
also, why is the default set to -160? I think the method would be less error-prone if there is no default; i.e. the argument is required.

Reply:
agreed